src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/css/libraryStyles.css

```
@@ -279,6 +279,18 @@ url('/webjars/form-flow/0.0.1/fonts/MaterialIcons-Regular.svg') format('svg');
   font-size: 1.6rem;
 }
 
+.list--checkmark li:before, .list--crossmark li:before {
+  content: '';
+  display: block;
+  position: absolute;
+  left: 0;
+  top: .25em;
+  width: 2.5rem;
+  height: 1.5rem;
+  background-image: url('/webjars/form-flow/0.0.1/images/emojis/checkmark.png');
+  background-repeat: no-repeat;
+  background-size: auto 1.5rem; }
```

Two things:
- `crossmark` should have it's own rule
- We only need to override the one property, all others will still be applied from `honeycrisp.min.css`

```suggestion
.list--checkmark li:before {
  background-image: url('/webjars/form-flow/0.0.1/images/emojis/checkmark.png');
}

.list--crossmark li:before {
  background-image: url('/webjars/form-flow/0.0.1/images/emojis/crossmark.png');
}
```